src/utils.jl

```diff
@@ -228,20 +228,45 @@ macro cplwarn(code, message)
     end
 end
 
+function showprogress(progress, message = "")
+    print("$(round(Int, progress)*100)..$message")
+    return true
+end
+
+function dummyprogress(progress, message = "")
```

Comment:
I think we should make this an internal function (I.e. `_dummyprogress`) and document it to always return `true`

Also, can we annotate `progress` with `::Cdouble`?

Reply:
Resolved by https://github.com/yeesian/ArchGDAL.jl/pull/353/commits/24c1304555c48b8e4931b9755e3113f35e312a52